resources/language/resource.language.fr_fr/strings.po

```diff
@@ -210,27 +210,27 @@ msgstr "Médiathèque"
 
 msgctxt "#30051"
 msgid "Profile set for playback from the library"
-msgstr ""
+msgstr "Profil utilisé lors de la lecture depuis la médiathèque"
 
 msgctxt "#30052"
 msgid "Set for library playback"
-msgstr ""
+msgstr "Utiliser ce profil lors de la lecture depuis la médiathèque"
 
 msgctxt "#30053"
 msgid "Remove for playback from library"
-msgstr ""
+msgstr "Ne plus utiliser ce profil lors de la lecture depuis la médiathèque"
```

Comment:
same here

Reply:
I removed the part "this profile" since it is quite obvious as the user opened the context menu on a profile so it should not confuse too much the user.
Do you think it is short enough now?

Reply:
yes it is good